Scripts/DefaultDebugMenuSummoner.cs

```diff
@@ -5,18 +5,53 @@ namespace DUCK.DebugMenu
 {
 	/// <summary>
 	/// A simple implementation of IDebugMenuSummoner, that summons the debug menu when F7 is pressed, or on a device,
-	/// when 6 fingers are pressed down
+	/// when 5 fingers are held down
 	/// </summary>
 	public class DefaultDebugMenuSummoner : MonoBehaviour, IDebugMenuSummoner
 	{
-		private const int TOUCH_AMOUNT = 6;
+		private const float TOUCH_TIME_OUT = 2;
+		private const int TOUCH_AMOUNT = 5;
 		private const KeyCode ACTIVATE_KEY = KeyCode.F7;
+		private float touchTime;
+		private int index;
 
 		public event Action OnSummonRequested;
 
+		private bool IsHeld()
+		{
+			var touches = Input.touches;
+
+			foreach (var touch in touches)
```

Comment:
I would consider splitting up the touch counting and the timer handling for readability. They would only be tiny methods, but it would help to split up the two conditions we need to check (how many held touches, and how long we're holding them).